lib/containers/utils.pm

```diff
@@ -41,52 +41,69 @@ sub test_seccomp {
     }
 }
 
+sub setup_registry {
+    my $runtime  = shift;
+    my $registry = get_var('REGISTRY');
+    if ($registry) {
+        assert_script_run('openssl s_client -servername $HOST -connect ' . $registry . ':443 < /dev/null 2>/dev/null | openssl x509 > /etc/pki/trust/anchors/certificate.crt');
+        assert_script_run('update-ca-certificates');
+        systemctl('restart docker') if ($runtime eq 'docker');
```

Comment:
Why not just `systemctl('restart $registry')` here?

Reply:
Assuming you mean `restart $runtime`, `podman`  doesn't have a service.

Reply:
I guess you mean `systemctl('restart $runtime')` but it doesn't work for podman, cause podman doesn't run as a service.